services/mesh.c

```diff
@@ -1290,6 +1290,22 @@ mesh_send_reply(struct mesh_state* m, int rcode, struct reply_info* rep,
 				&r->edns, &r->query_reply, m->s.region, &r->start_time))
 					r->edns.opt_list_inplace_cb_out = NULL;
 		}
+		/* Send along EDE BOGUS EDNS0 option when answer is bogus */
+		if(rcode == LDNS_RCODE_SERVFAIL &&
+			m->s.env->need_to_validate && (!(r->qflags&BIT_CD) ||
+			m->s.env->cfg->ignore_cd) && rep &&
```

Comment:
Should an EDE code be included when checking is disabled?
I tried with quad1 which is the only public resolver supporting EDE I know of, and they ignore cd bit!
Maybe we should seek advice at DNSOP?

Reply:
> Maybe we should seek advice at DNSOP?
I like this idea!